src/core/Cline.ts

```diff
@@ -135,6 +137,9 @@ export class Cline {
 		this.browserSession = new BrowserSession(provider.context, browserSettings)
 		this.diffViewProvider = new DiffViewProvider(cwd)
 		this.customInstructions = customInstructions
+		this.preferredLanguage = getLanguageKey(
```

Comment:
I dont want to include these instructions if the user is by default using english. perhaps we check if english then not pass in these instructions?